src/basic/log.c

```diff
@@ -1445,6 +1445,54 @@ static const char *const log_target_table[_LOG_TARGET_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(log_target, LogTarget);
 
+void log_task_chain(pid_t pid, const char *special_action_string) {
+        pid_t ppid;
+        uid_t uid;
+
+        const int log_facility_saved = log_facility;
+        const LogTarget log_target_saved = log_get_target();
+        log_set_facility(LOG_AUTHPRIV);
+        log_set_target(LOG_TARGET_SYSLOG);
+        log_open();
```

Comment:
That is very strange. Why would you want to fiddle with log settings here?

Reply:
I would like this to go to a /var/log/secure or wherever syslog is configured to log authpriv information.

Reply:
Done.